README.md

```diff
@@ -43,11 +44,11 @@ const [Map] = await loadModules(['esri/map'], options);
 
 ## TypeScript typings for ES modules
 
-Typings are included with the npm install of `@arcgis/core@next` ES modules. For most use cases, no special TypeScript configurations are required.
+Typings are included with the npm install of ES modules. For most use cases, no special TypeScript configurations are required.
 
 ## TypeScript typings for AMD modules
 
-The typings are also included with `arcgis-js-api@next` AMD modules, and they can be used in two ways.
+The typings are also included for AMD modules, and they can be used in two ways.
```

Comment:
For consistency: "The typings are also included with the npm install of AMD modules, ..."